scripts/gen_coverage_report.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Compute code coverage for the following steps:
+#
+# - unit tests
+# - snapshot dowloading
+# - validating 200 tipsets from the snapshot
+# - syncing to HEAD
+# - exporting a snapshot
+#
+# llvm-cov can be installed by running: cargo install cargo-llvm-cov
+#
+
+set +e
+
+cargo llvm-cov --workspace clean
+cargo llvm-cov --workspace --no-report --features slow_tests
+cargo llvm-cov --no-report run --bin=forest-cli -- --chain calibnet db clean --force
+cargo llvm-cov --no-report run --bin=forest-cli -- --chain calibnet snapshot fetch -s .
+# SC2012 says to use 'find' instead of 'ls' but I don't know how to do that.
```

Comment:
`find . -name "*.car"` doesn't work? I don't think we should expect multiple CAR files.

Reply:
There might be more car files when running the script locally. Is there an easy way to sort by modification time with `find`?

Reply:
None I am aware of, only the `print` voodoo, which doesn't help with readability. Alternatively, we could save the snapshot to a temporary directory (personally, that's what I would prefer) with `mktemp`.